Movement changes

SUBERNER opened the pull request:
Created Player 2.0 and in prefabs and changed player 1.0 for 2.0

Created new movement script.
Fixed jump problem
fixed slope problem

added composite collider to slopes

SUBERNER committed:
Updated Player controller version 2.0

Created brand new player prefab with updated controls and better movement.

Prefab is named player 2.0
fixed slope movement
fixed acceleration
fixed jump force
other small changes

SUBERNER committed:
Merge branch 'test-branch' of https://github.com/USA-Video-Game-Development-Club/VGDC-CampusRush into test-branch

SUBERNER committed:
Merge branch 'test-branch' of https://github.com/USA-Video-Game-Development-Club/VGDC-CampusRush into test-branch

firestar1290 reviewed (changes requested):
Please don't include changes to the scenes in your pull request

SUBERNER committed:
Merge branch 'test-branch' of https://github.com/USA-Video-Game-Development-Club/VGDC-CampusRush into test-branch

SUBERNER committed:
Player movement and collectable

Finished player movement
Smooth acceleration
sprint
variable jump

Added temp collectable
added double jump collectable

added level temp maker

SUBERNER committed:
Movement changes

SUBERNER committed:
Merge branch 'test-branch' into test-branch
